plugins/source/stripe/resources/services/invoices/invoice_items_test.go

```diff
@@ -0,0 +1,18 @@
+package invoices_test
+
+import (
+	"testing"
+
+	"github.com/cloudquery/cloudquery/plugins/source/stripe/client"
+	"github.com/cloudquery/cloudquery/plugins/source/stripe/resources/services/invoices"
+)
+
+func TestInvoicesInvoiceItem(t *testing.T) {
+	tbl := invoices.InvoiceItems()
+	for i, c := range tbl.Columns {
+		if c.Name == "plan" {
+			tbl.Columns[i].IgnoreInTests = true
```

Comment:
We need to do this because the mock server doesn't return the column even though the fixture is populated correctly.

Reply:
Why isn't the column getting returned - isn't that something we control?

Reply:
No - mock server is implemented in the https://github.com/stripe/stripe-mock package and we're merely utilizing it to handle our requests, rather than faking our own server.